### PR TITLE
Make explorer search case insensitive

### PR DESCRIPTION
**Pull request by dimroc**
https://www.pivotaltracker.com/story/show/165476758

### CODE REVIEW

File: explorer/src/migration/1556119396403-ConvertJobRunSearchableColsToCitext.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    await queryRunner.query(`
+      DROP INDEX job_run_chainlink_node_id_run_id_idx;
+      DROP INDEX job_id_idx;
+      DROP INDEX job_run_requester_idx;
+      DROP INDEX job_run_request_id_idx;
+      DROP INDEX job_run_tx_hash_idx;
+    `)
+
+    await queryRunner.query(`
+      ALTER TABLE "job_run" ALTER COLUMN "runId" TYPE citext;
```
Comment (j16r) [inline explorer/src/migration/1556119396403-ConvertJobRunSearchableColsToCitext.ts]: I'm pretty certain this is not compatible with sqlite, you have to do the create new table COPY across trick.

Comment (dimroc) [reply explorer/src/migration/1556119396403-ConvertJobRunSearchableColsToCitext.ts]: @j16r This is reserved just for `explorer` so compatibility with sqlite isn't necessary.

Comment (dimroc) [reply explorer/src/migration/1556119396403-ConvertJobRunSearchableColsToCitext.ts]: SQLite doesn't support `citext` at all.

Comment (j16r) [reply explorer/src/migration/1556119396403-ConvertJobRunSearchableColsToCitext.ts]: Ahhh! Well carry on then ;)

**Pull request merged**